### ISSUE TITLE
No 5.1 output on HDMI, with release 1.24

### ISSUE THREAD

**Issue by hervald** [avafinger/nanopi-m4-ubuntu-base-minimal#30]
hello,
I tried your release 1.24 and I did not get 5.1 sound with kodi over HDMI. In fact, my AV amplifier is always receiving a stereo pcm signal. I tried to read the video with passthrough, and it does not work too. I read all day long how to configure kodi, pulseaudio, ... but it's a chess. 
It is not a problem from my AV : I can perfectly read a video with VLC on my laptop (with the same hdmi cable), either using passthrough or configuring my sound output to be HDMI 5.1. In both cases, I really have 5.1 sound out of my speakers.

In conclusion, I would like to know if you have the same behavior, so if it is normal. Or if I have something badly installed, configured, ... And in this case, have you got an idea about what could be wrong.

Thanks

**Comment by avafinger** [comment]
i am not an audiophile, but i think you mean "enable passthrough" in kodi. It's been a long time since I don't play with kodi but I heard people had a hard time enabling pass-through on rk3399. I still believe it is PulseAudio config related, but I can be wrong.
For me, I just plugged hdmi and had hdmi sound output, that was it, but I think that is not just what you want.
I don't know if it helps you or not.
